src/pivotal-ui/components/links/links.scss

```diff
@@ -49,30 +48,32 @@ a{
   &:hover, &:focus {
     cursor: pointer;
     text-decoration: none;
+    color: lighten($blue-2, 5%);
+  }
+
+  &.active{
+    color: darken($blue-2, 5%);
   }
 
   &.link-text {
-    &:not(.link-lowlight):hover {
```

Comment:
@kennyw12 was this change here enough to fix hover styles for **all** links? I know that bootstrap by default has an underline on links. Did you inspect all the links to ensure the hover style was fixed for everything?